frontends/web/src/routes/account/send/send-wrapper.tsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
This file is a wrapper for our `send` component **and it's a temporary solution / file**.

Reason: `send` needs something from the context store, but since it's still not a functional cmpt yet, we use this wrapper to get it for us (`useContext(RatesContext`) which passes it down to  our actual `send` component as props.